packages/react-component-playground/src/components/welcome-screen.less

```diff
@@ -0,0 +1,42 @@
+@anchor-color: #08c;
+@anchor-color-focus: #005580;
+@text-color: #545454;
+
+.welcome-screen {
+  position: absolute;
+  left: @left-nav-width;
+  min-height: 100vh;
+  width: calc(100% ~"-" @left-nav-width);
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  background: @background;
+
+  &-inner {
+    font-family: @default-font-family;
+    width: 50%;
+    margin: auto;
+    padding: 16px;
+    color: @text-color;
+    background-color: #ffffff;
```

Comment:
I'd go with `rgba(255, 255, 255, 0.9)`, tweaking the alpha to make sure the text is readable. We need to see how it looks, though.